backoffice/middleware.py

```diff
@@ -0,0 +1,32 @@
+# _*_ coding: utf8 _*_
+"""Disabling user impersonation on admin and backoffice paths."""
+
+import re
+
+from masquerade.middleware import MasqueradeMiddleware
+
+MASQUERADE_WHITELIST = map(
+    re.compile, (
+        r'^/backoffice/.*',
+        r'^/admin/.*',
```

Comment:
Do you think URLs must be hard coded? Why not "resolving" their namespace?

Reply:
we can't resolve because these are patterns, englobing many namespaces. cc @rmoch